content/community/organization/policies.html

```diff
@@ -38,13 +38,28 @@ <h4>Coding Guidelines</h4>
 <HR>
 <h2>Communication</h2>
 
+<h4>General rules</h4>
+
+<p>Be nice to other people. Avoid insults and personal attacks.
+In case someone has innapropriate behavior, raise the issue to members of the
+Haiku development team, or at the last resort get in touch with contact@haiku-inc.org .
```

Comment:
comma after `resort`, and make the email a bit more "obscure" (i.e. `contact at <a href="haiku-inc.org">haiku-inc.org</a>`).